package.json

```diff
@@ -722,6 +722,21 @@
           "markdownDescription": "`#editor.lineNumbers#` is determined by the active Vim mode, absolute when in insert mode, relative otherwise.",
           "default": false
         },
+        "vim.smartQuotes.enable": {
+          "type": "boolean",
+          "markdownDescription": "Enable smart quotes movements (similar to targets.vim)",
+          "default": true
+        },
+        "vim.smartQuotes.breakThroughLines": {
+          "type": "boolean",
+          "markdownDescription": "Whether to break through lines when using [n]ext/[l]ast motion",
+          "default": true
+        },
+        "vim.smartQuotes.aIncludesSurrouningSpaces": {
+          "type": "boolean",
+          "markdownDescription": "Whether to use default vim behaviour when using `a` (e.g. da') which include surrounding spaces, or not, as for other text objects.",
```

Comment:
Capitalize `vim` and add backticks around `da'`

Reply:
```suggestion
          "markdownDescription": "Whether to use default Vim behavior when using `a` (e.g. `da'`) which include surrounding spaces, or not, as for other text objects.",
```